docs/language-usage.rst

```diff
@@ -0,0 +1,354 @@
+               }
+             }
+           ]
+         },
+         {
+           'name': 'Italian',
+           'type': 'LOCATION',
+           'metadata': {
+             'wikipedia_url': 'http://en.wikipedia.org/wiki/Italy'
+           },
+           'salience': 0.20193423,
+           'mentions': [
+             {
+               'text': {
+                 'content': 'Italian',
+                 'beginOffset': 25
+               }
+             }
+           ]
+         },
+         {
+           'name': 'The Calling of Saint Matthew',
+           'type': 'WORK_OF_ART',
+           'metadata': {
+             'wikipedia_url': 'http://en.wikipedia.org/wiki/index.html?curid=2838808'
+           },
+           'salience': 0.03863598,
+           'mentions': [
+             {
+               'text': {
+                 'content': 'The Calling of Saint Matthew',
+                 'beginOffset': 56
+               }
+             }
+           ]
+         }
+       ],
+       'language': 'en'
+     }
+
+In addition, the encoding of the text in the document can be
+specified in the request. By default, the UTF-8 is used:
+
+  .. code-block:: python
+
+     >>> document.analyze_entities(encoding=language.Encoding.UTF8)
+
+but the encoding can be one of
+:attr:`Document.UTF8 <gcloud.language.document.Encoding.UTF8>`,
+:attr:`Document.UTF16 <gcloud.language.document.Encoding.UTF16>`, or
+:attr:`Document.UTF32 <gcloud.language.document.Encoding.UTF32>`.
+
+Analyze Sentiment
+-----------------
+
+The :meth:`~gcloud.language.document.Document.analyze_sentiment` method
+analyzes the sentiment of the provided text. Currently, this method
+only supports English text.
+
+  .. code-block:: python
+
+     >>> text_val = "Jogging isn't very fun."
+     >>> document = client.document(text_val)
+     >>> document.analyze_sentiment()
+     {
+         'documentSentiment': {
+             'polarity': -1,
+             'magnitude': 0.8,
+         },
+         'language': 'en',
+     }
+
+Annotate Text
+-------------
+
+The :meth:`~gcloud.language.document.Document.annotate_text` method
+analyzes a document and provides a full set of text annotations,
+including semantic, syntactic, and sentiment information. This API
+is intended for users who are familiar with machine learning and
+need in-depth text features to build upon.
+
+The API can be used to extract any subset of `Features`_ among
+:attr:`~gcloud.language.document.Features.SYNTAX`,
+:attr:`~gcloud.language.document.Features.ENTITIES`, and
+:attr:`~gcloud.language.document.Features.DOCUMENT_SENTIMENT`.
+
+  .. code-block:: python
+
+     >>> text_val = 'The cow jumped over the moon.'
+     >>> document = client.document(text_val)
+     >>> features = [
+     ...     language.Features.SYNTAX,
+     ...     language.Features.ENTITIES,
+     ...     language.Features.DOCUMENT_SENTIMENT,
+     ... ]
+     >>> document.annotate_text(features=features)
+     {
+         'tokens': [
```

Comment:
As above, this was just the JSON payload returned by the API. As suggested by @tseaver I am wrapping them in native object.

>  we're supposed to let people use dot syntax to mess with stuff, not dictionary syntax.

"supposed to" since when?